modules/debts_api/app/models/debts_api/v0/digital_dispute_submission.rb

```diff
@@ -62,6 +74,53 @@ def extract_debt_types(disputes)
       def extract_dispute_reasons(disputes)
         disputes.map { |d| d[:dispute_reason] }.compact.uniq
       end
+
+      def send_success_email
+        StatsD.increment("#{STATS_KEY}.send_success_email.enqueue")
+        user = User.find(user_uuid)
+        return if user&.email.blank?
+
+        DebtsApi::V0::Form5655::SendConfirmationEmailJob.perform_async(
+          {
+            'submission_type' => 'digital_dispute',
+            'email' => user.email,
+            'first_name' => user.first_name,
+            'user_uuid' => user.uuid,
+            'template_id' => CONFIRMATION_TEMPLATE
+          }
+        )
+        StatsD.increment("#{STATS_KEY}.send_success_email.success")
```

Comment:
I'm not sure this is a good place for this since the job is async and could fail right?